.github/workflows/tests.yml

```diff
@@ -218,6 +218,98 @@ jobs:
       - name: "Run the tests"
         run: /build/faasm/bin/tests
 
+  sgx-tests:
```

Comment:
I don't merge this with `tests` as a single job with a matrix because `tests` already has the `sanitisers` matrix, and we only want to run the SGX tests with no sanitisers.